inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/graph/KBQualifier.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2018
+ * Ubiquitous Knowledge Processing (UKP) Lab
+ * Technische Universität Darmstadt
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package de.tudarmstadt.ukp.inception.kb.graph;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.cyberborean.rdfbeans.datatype.DatatypeMapper;
+import org.cyberborean.rdfbeans.datatype.DefaultDatatypeMapper;
+import org.eclipse.rdf4j.model.BNode;
+import org.eclipse.rdf4j.model.IRI;
+import org.eclipse.rdf4j.model.Literal;
+import org.eclipse.rdf4j.model.Statement;
+
+public class KBQualifier
+    implements Serializable
+{
+    private static final long serialVersionUID = 4648563545691138244L;
+
+    private KBStatement kbStatement;
+
+    private String language;
+
+    private KBHandle kbProperty;
+
+    private Object value;
+
+    private List<Statement> originalStatements;
+
+    public KBQualifier(KBStatement kbStatement, KBHandle kbProperty, Object aValue)
+    {
+        this.kbStatement = kbStatement;
+        this.kbProperty = kbProperty;
+        DatatypeMapper mapper = new DefaultDatatypeMapper();
+
+        if (aValue instanceof Literal) {
```

Comment:
Ist das von woanders dupliziert (KBStatement)? Ich würde es sonst wie den ValueMapper an einer Stelle machen

Reply:
They have a slight difference. Besides, this code may also set language and set value, so I cannot just return the value.
```
if (aValue instanceof Literal) {
            Literal litValue = (Literal) aValue;
            language = litValue.getLanguage().orElse(null);
            value = mapper.getJavaObject(litValue);
        }
```